lib/capability/capability.go

```diff
@@ -48,6 +48,15 @@ func SetCapabilities(config *configv1.ClusterVersion,
 	return capabilities
 }
 
+// GetCapabilitiesAsMap returns the slice of capabilities as a map with default values.
+func GetCapabilitiesAsMap(capabilities []configv1.ClusterVersionCapability) map[configv1.ClusterVersionCapability]struct{} {
```

Comment:
nit, you can use this helper to seed an initial `enabled` set [here][1].

[1]: https://github.com/openshift/cluster-version-operator/blob/910117429c2ceef37ec7cc25b64bfa7a0058b9da/lib/capability/capability.go#L154-L156